server/svix-server/src/queue/redis.rs

```diff
@@ -144,11 +165,14 @@ impl TaskQueueSend for RedisQueueProducer {
         let key = to_redis_key(&delivery);
         if let Some(timestamp) = timestamp {
             let _: () = pool
-                .zadd(DELAYED, key, timestamp.timestamp())
```

Comment:
Looking at the worst experience here everywhere by using `redis::Cmd::zadd` instead of `pool.zadd`, I wonder if we should have just had our own wrappers for these functions instead of making the experience worst everywhere.

Can probably easily just implement it with a macro.